### ISSUE TITLE
Welcome:  "Customize" heading read out as list

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#51986]
**Environment Details:**
VSCode Version : 1.23.1
OS Version : Win10

**Additional Details:**
MAS Violated : MAS 1.3.1
ScreenReader Versions : NVDA2018.1.1 / JAWS18.0.5038

**Repro Steps:**
1. Launch VS Code while NVDA is ON.
2. Navigate to Welcome screen.
3. Read all the content present under the heading "Customize".

**Actual:**
NVDA reads content of -"Tools and Languages ...", "Settings and Keybindings.." as list. These are headings and only links present in between the content can be considered in a list. Unnecessary markup of lists is present for static content.

**Expected:**
NVDA should read the headings seperately and list markup should be removed for the static content present.
Lists can be formed out of links given inside above mentioned headings.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
The user will come across unnecessary markup of lists for the static content present on the page which is not actionable.

**MAS Reference:**
MAS 1.3.1 - https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment For Reference:**
[11y_VSCode_WelcomeScreen_NVDA_MarkupIssue.pptx](https://github.com/Microsoft/vscode/files/2105378/11y_VSCode_WelcomeScreen_NVDA_MarkupIssue.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
This feels like the weclome page should be restructured. Leaving to @chrmarti to decide

**Comment by isidorn** [comment]
Assigning to October so we discuss if something can be done here.
